source/extensions/filters/network/dubbo_proxy/active_message.cc

```diff
@@ -81,7 +81,7 @@ FilterStatus ActiveResponseDecoder::applyMessageEncodedFilters(MessageMetadataSh
   auto status = parent_.applyEncoderFilters(
       nullptr, ActiveMessage::FilterIterationStartState::CanStartFromCurrent);
   switch (status) {
-  case FilterStatus::StopIteration:
+  case FilterStatus::PauseIteration:
```

Comment:
May be keep the `StopIteration`?

Reply:
Sure, we can keep StopIteration as it is for back-compatible.